var/spack/repos/builtin/packages/amrvis/package.py

```diff
@@ -50,8 +50,10 @@ class Amrvis(MakefilePackage):
         multi=False,
         description='Floating point precision'
     )
-    variant('mpi', default=True, description='Enable MPI parallel support')
+    variant('mpi', default=False, description='Enable MPI parallel support')
```

Comment:
I agree that almost all packages are set to MPI on by default so I guess I don't really have a strong opinion about this and I will change it back, but I just think most people who install this would just want `~mpi` for simplicity.